vespalib/src/vespa/vespalib/data/slime/array_value.h

```diff
@@ -28,14 +28,15 @@ class ArrayValue : public Value
     }
 
 public:
-    ArrayValue(SymbolTable &table, Stash & stash) : _symbolTable(table), _stash(stash), _values() {}
+    ArrayValue(SymbolTable &table, Stash & stash);
     ArrayValue(const ArrayValue &) = delete;
     ArrayValue &operator=(const ArrayValue &) = delete;
 
     Type type() const override { return ARRAY::instance; }
     size_t children() const override { return _values.size(); }
     size_t entries() const override { return _values.size(); }
     void traverse(ArrayTraverser &at) const override;
+    void reserve(size_t sz) { _values.reserve(sz); }
```

Comment:
maybe put this with the ctor/assignment operators and not in the middle of the Value interface implementation.

Reply:
Done